package.json

```diff
@@ -10,6 +10,9 @@
     "build": "yarn workspace @maggie-and-dustin/wedding run build",
     "start": "yarn workspace @maggie-and-dustin/wedding run start"
   },
+  "resolutions": {
+    "graphql": "^14.7.0"
```

Comment:
This causes an *extremely* confusing bug, notably:

```

 ERROR 

Missing onError handler for invocation 'building-schema', error was 'Error: ContentfulBlogPost.gatsbyPath provided incorrect OutputType: 'String''. Stacktrace was 'Error: ContentfulBlogPost.gatsbyPath provided incorrect OutputType: 'String'
    at TypeMapper.convertOutputFieldConfig (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/graphql-compose/lib/TypeMapper.js:294:15)
    at resolveOutputConfigAsThunk (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/graphql-compose/lib/utils/configAsThunk.js:19:41)
    at ObjectTypeComposer.getFieldConfig (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/graphql-compose/lib/ObjectTypeComposer.js:300:58)
    at /Users/dschau/Code/Personal/maggieanddustin.com/node_modules/graphql-compose/lib/utils/toInputObjectType.js:44:19
    at Array.forEach (<anonymous>)
    at toInputObjectType (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/graphql-compose/lib/utils/toInputObjectType.js:38:14)
    at ObjectTypeComposer.getInputTypeComposer (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/graphql-compose/lib/ObjectTypeComposer.js:600:84)
    at getSortInput (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/gatsby/src/schema/types/sort.ts:152:42)
    at addTypeToRootQuery (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/gatsby/src/schema/schema.js:1107:23)
    at processTypeComposer (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/gatsby/src/schema/schema.js:218:13)
    at async Promise.all (index 41)
    at updateSchemaComposer (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/gatsby/src/schema/schema.js:168:3)
    at buildSchema (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/gatsby/src/schema/schema.js:62:3)
    at build (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/gatsby/src/schema/index.js:105:18)
    at buildSchema (/Users/dschau/Code/Personal/maggieanddustin.com/node_modules/gatsby/src/services/build-schema.ts:19:3)'
```

Reply:
```
maggieanddustin.com|unified-routes⚡ ⇒ yarn why graphql
yarn why v1.22.5
[1/4] 🤔  Why do we have the module "graphql"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "graphql@14.6.0"
info Has been hoisted to "graphql"
info Reasons this module exists
   - "workspace-aggregator-9e31b3c6-21de-491f-b529-aef9271e51c6" depends on it
   - Hoisted from "_project_#@maggie-and-dustin#wedding#gatsby#graphql"
   - Hoisted from "_project_#@maggie-and-dustin#wedding#gatsby#gatsby-cli#gatsby-recipes#graphql"
info Disk size without dependencies: "3.22MB"
info Disk size with unique dependencies: "3.32MB"
info Disk size with transitive dependencies: "3.32MB"
info Number of shared dependencies: 1
=> Found "gatsby-plugin-page-creator#graphql@14.7.0"
info This module exists because "_project_#@maggie-and-dustin#wedding#gatsby#gatsby-plugin-page-creator" depends on it.
info Disk size without dependencies: "3.11MB"
info Disk size with unique dependencies: "3.21MB"
info Disk size with transitive dependencies: "3.21MB"
info Number of shared dependencies: 1
✨  Done in 0.86s.
```